software/authbox/badgereader_wiegand_gpio.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2018 Ace Monster Toys. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Wiegand based badge reader directly connected via GPIO
+"""
+
+from authbox.api import BaseWiegandPinThread, GPIO
+import time
+
+
+class WiegandGPIOReader(BaseWiegandPinThread):
+  """Badge reader hardware abstraction.
+
+  A Wiegand GPIO badge reader is defined in config as:
+
+    [pins]
+    name = WiegandGPIOReader:3:5
+
+  where 3 is the D0 pin (physical numbering), and 5 is the D1 pin (also 
```

Comment:
I know these are kind of minimal examples, but could you make it use the pins from J5?  When you add more docs, this should link to there.